src/licensedcode/data/licenses/jpl-image.LICENSE

```diff
@@ -0,0 +1,71 @@
+---
+key: jpl-image
+short_name: JPL Image Use Policy
+name: JPL Image Use Policy
+category: Source-available
```

Comment:
Is this category: `Source Available` right? Also this is for images not software?

Reply:
@DennisClark ^

Reply:
@DennisClark ^